cdap-docs/developer-manual/source/pipelines/creating-pipelines.rst

```diff
@@ -212,6 +212,74 @@ Two interfaces are available:
   underlying ``cron`` program. The details of that program will depend on the operating
   system used by the host of the CDAP Master process.
 
+With the CDAP Studio, you can also create a schedule that launches a pipeline when another pipeline
+reaches some certain statuses, and let the scheduled pipeline use properties from the triggering
```

Comment:
list out what is possible and split into two sentences.

when another pipeline completes, fails, <others>.  Arguments and plugin properties from the triggering pipeline can be passed to the triggered pipeline as arguments.